### PR TITLE
Fix billing

### LINKED ISSUE #111: Biller is broken
Few large issues with the biller currently that have it led to it being rolled back:
1. If there isn't a container ID for a specific app (e.g. isn't running / deployed) the biller should just not bill the app instead of killing the entire service.
2. When an app is deployed the biller continues to use the old container ID instead of switching to the new one. Fixing this likely involves a post-deploy hook.

### PR DESCRIPTION
**Pull request by anirudhb**
Closes #111 

Oh yeah haven't tested any of this, should probably do that.

### CODE REVIEW

File: dokku_plugin/subcommands/cid/cid.go [hunk 1]
```diff
@@ -25,14 +25,14 @@ func main() {
...
-		os.Exit(1)
+		fmt.Println()
+		os.Exit(0)
 	}
 
 	contents, err := ioutil.ReadAll(file)
 	if err != nil {
-		fmt.Println("Error reading container file")
-		os.Exit(1)
+		fmt.Println()
```
Comment (cjdenio) [inline dokku_plugin/subcommands/cid/cid.go]: This should still trigger a non-zero exit code in my opinion

**Review by cjdenio — CHANGES_REQUESTED** [review]
please don't hate me :rayge:

File: dokku_plugin/subcommands/cid/cid.go (hunk 1 above)
Comment (anirudhb) [reply dokku_plugin/subcommands/cid/cid.go]: Fair

**Review by cjdenio — APPROVED** [review]
hang on to your hats

**Pull request merged**